shared/fixes/bash/accounts_authorized_local_users_sidadm_orasid.sh

```diff
@@ -7,31 +7,31 @@ default_os_user="root"
 
 # add users sidamd, orasid, sapadm and oracle if needed
 userlist="root"
+sapmnt_SID_stem="/sapmnt/[A-Z][A-Z0-9][A-Z0-9]"
 
 # if /sapmnt is a directory or a symbolic link to a directory,
 # then try to add SAP system users to the userlist
 if [ -d "/sapmnt" ] ; then 
 	# if /sapmnt/SID exists, add sidadm to the userlist
-	sapmntSIDlist=$(find /sapmnt/ -regex '^/sapmnt/[A-Z][A-Z0-9][A-Z0-9]$')
-	for i in $sapmntSIDlist ; do
+	path_sapmnt_SID_list=$(find /sapmnt/ -regex "^$sapmnt_SID_stem$")
+	for i in $path_sapmnt_SID_list ; do
```

Comment:
I would suggest renaming `i` to something like `path_including_SID`, so it is also more obvious what is the intent behind `${...:8:3}`.